### PR TITLE
Move events repository here

### PR DESCRIPTION
**Pull request by RDIL**
Checklist:
- [x] Base files
- [X] Includes
- [x] Layouts
- [x] Data
- [x] Posts

### CODE REVIEW

**Comment by RDIL** [comment]
@jancborchardt would you be ok with leaving the events before 2016 out?

**Comment by jancborchardt** [comment]
> @jancborchardt would you be ok with leaving the events before 2016 out?

Since it shows the development of the collective and how long it has been around, we should definitely have all content and all events in.

**Comment by jancborchardt** [comment]
@RDIL is there a specific issue with pre-2016 events, or how come you ask? :)

**Comment by RDIL** [comment]
Nope, just thought they were a bit old but I’ll put them in

File: _config.yml [hunk 1]
```diff
@@ -34,5 +41,16 @@ collections:
...
     style: compressed
 
+# for events
+status: &status
+  - name: "Upcoming"
+    value: "upcoming"
+  - name: "Started"
+    value: "started"
+  - name: "Past"
+    value: "Past"
```
Comment (evalica) [inline _config.yml]: value is "past", small "p"

Comment (RDIL) [reply _config.yml]: Thanks, fixing now

**Comment by RDIL** [comment]
@jancborchardt @evalica We should be done

**Comment by jancborchardt** [comment]
@AnXh3L0 @dmichl @ei8fdb @evalica @Jc-L @simonv3 could some of the Jekyll experts check this out? :)

@RDIL it would be cool if you can squash the commits into one. (It’s also fine if not, we will then just squash on merging. :)

**Comment by RDIL** [comment]
Hey @jancborchardt can OSD also consider moving on to [Netlify](https://netlify.com)?

Its free, connects to GitHub, and creates deploy previews of pull requests so you can see what it will look like once the PR is merged.

**Comment by jancborchardt** [comment]
@RDIL we discussed having something like Netlify in the past, but we’d like to not introduce more stuff to manage. GitHub Pages is quite ok for our use-case, and we are not locked-in to a specific provider. (e.g. we could switch to GitLab pages almost instantly).

@AnXh3L0 could you check this pull request out too so we can solve this before jumping into the redesign? :) (ref https://github.com/opensourcedesign/opensourcedesign.github.io/issues/242 )

**Comment by AnXh3L0** [comment]
@jancborchardt I will check this PR today.

**Comment by AnXh3L0** [comment]
I checked the PR and most pages, things are looking ok, approved from my side.

**Comment by AnXh3L0** [comment]
@RDIL Is there anything I'm missing about the Events page showing "Page not found" when I build it locally? Otherwise everything else seems ok to me.

**Comment by RDIL** [comment]
I can't check now, but I will later

**Comment by elioqoshi** [comment]
It's a bit of a pity to start a PR and abandon it since the new website work was based on this as well. It would be great if someone could take a look at this since we are kind of stuck.

**Comment by jancborchardt** [comment]
@RDIL @vinitshahdeo @AnXh3L0 @evalica @elioqoshi I’m back from vacation now too. Is there any specific list of issues that are left to fix? Let’s split it into blockers we need to fix before merging, and things which can wait for polishing.

**Comment by RDIL** [comment]
I will try to test it out so we can move on from there

**Comment by RDIL** [comment]
Okay everyone, I just tested it and it is now working.
Screenshot 2019-10-26 at 10 30 50 AM

**Comment by jancborchardt** [comment]
@RDIL nice! :) There’s at least the 2020 FOSDEM event which has come after this, could you add that on top and check if others are new?

**Comment by RDIL** [comment]
Yup, its ordered by date - latest on top.  We should be good to go.

**Comment by jancborchardt** [comment]
Wrestling a bit with my local Ruby setup before I can actually test … only looking at the code, I was thinking about these things:
- Doesn’t `_posts/events` duplicate what’s in `events`?
- With the events being in `_posts`, don’t they conflict with the articles?

Also cc @AnXh3L0 @ei8fdb for testing :)

**Comment by RDIL** [comment]
Going to merge on Thursday if no other feedback is provided.

**Comment by RDIL** [comment]
Actually it might be better to make events into a collection, so I'm going to try that.

**Comment by jancborchardt** [comment]
> * Doesn’t `_posts/events` duplicate what’s in `events`?
> * With the events being in `_posts`, don’t they conflict with the articles?

@RDIL could you answer those? :) Also, is this ready to review now or still something to do?

**Comment by RDIL** [comment]
Putting finishing touches on :)

**Comment by RDIL** [comment]
Okay and were ready.  Had to make a few unexpected changes to fix some bugs, but other then that no too major changes.
Have fun reviewing 9000 lines of code changed haha

**Comment by RDIL** [comment]
To answer your questions @jancborchardt,
1. No, I reconfigured it so it won't, but the GH pages will need to be disabled on the events repo a few minutes before this is merged.
2. I moved the events to their own Jekyll collection in a previous commit somewhere, I can assure you events and posts won't conflict.

**Comment by RDIL** [comment]
Tip for reviewers: you can just ignore the _events stuff since its just a copy of the existing markdown files.

**Comment by jancborchardt** [comment]
@AnXh3L0 @dmichl and others, did you have time to look at this as well? :)

**Comment by RDIL** [comment]
We just need one more review!

**Comment by ei8fdb** [comment]
Sorry, I don't know enough about Ruby!

**Comment by RDIL** [comment]
@jancborchardt the events repo needs GitHub pages disabled from settings and to be archived before this can be merged.

**Comment by RDIL** [comment]
@AnXh3L0 since your online and have owner permissions can you please do [the steps I listed above](https://github.com/opensourcedesign/opensourcedesign.github.io/pull/239#issuecomment-615370916)?  It will take a minute and is the only thing we need to get this merged!

**Comment by AnXh3L0** [comment]
Let me check @RDIL, I will comment here in a few minutes.

**Comment by AnXh3L0** [comment]
@RDIL we need to make a few more fixes, the link in the homepage for "See all events" takes you to /events, meanwhile the events list itself is on /events/.

**Comment by RDIL** [comment]
@AnXh3L0 creating a follow-up PR now.

**Pull request merged**